maptk/local_geo_cs.cxx

```diff
@@ -135,7 +135,7 @@ ::update_metadata(vital::simple_camera const& cam,
   vital::vector_3d c = cam.get_center();
   vital::geo_point gc(vector_2d(c.x(), c.y()) + geo_origin_.location(),
                       geo_origin_.crs());
-  vector_2d lon_lat = gc.location(SRID::lat_lon_WGS84);
+  geo_point lon_lat = geo_point(gc.location(SRID::lat_lon_WGS84),gc.crs());
```

Comment:
It looks like this was left over from our conversion to using the new geo_point data structure.  I think we might be able to bypass the lon_lat variable altogether now by directly passing `gc` in line 140.  Is that right @mwoehlke-kitware?  I think as it is this change is problematic because it is converting the point to lat_lon_WGS84 (from UTM in this case) and then creating a new geopoint with the lat & lon values but with the UTM coordinate space used by geo_origin_.

Reply:
>  I think we might be able to bypass the lon_lat variable altogether now by directly passing gc in line 140.

That should work, yes.

>  I think as it is this change is problematic because it is converting the point to lat_lon_WGS84 (from UTM in this case) and then creating a new geopoint with the lat & lon values but with the UTM coordinate space used by geo_origin_.

Yes, this is correct; as-is, the second parameter of the ctor should be `SRID::lat_lon_WGS84`, not `gc.crs()`.